package.json

```diff
@@ -120,6 +145,18 @@
           "when": "view == CompilerToolchainView",
           "group": "navigation"
         }
+      ],
+      "editor/title": [
```

Comment:
for reviewer, https://code.visualstudio.com/api/references/contribution-points#menu-example